Docker.DotNet/Microsoft.Net.Http.Client/UnixDomainSocketEndPoint.cs

```diff
@@ -0,0 +1,110 @@
+#if NETSTANDARD1_6
+
+using System;
+using System.Runtime.InteropServices;
+using System.Diagnostics;
+using System.Text;
+using System.Net;
+using System.Net.Sockets;
+
+namespace Microsoft.Net.Http.Client
+{
+    internal sealed class UnixDomainSocketEndPoint : EndPoint
+    {
+        private const AddressFamily EndPointAddressFamily = AddressFamily.Unix;
+
+        private static readonly Encoding s_pathEncoding = Encoding.UTF8;
+
+        private static readonly int s_nativePathOffset;
+
+        private static readonly int s_nativePathLength;
+
+        private static readonly int s_nativeAddressSize;
+
+        private readonly string _path;
+
+        private readonly byte[] _encodedPath;
+
+        [DllImport("System.Native", EntryPoint = "SystemNative_GetDomainSocketSizes")]
+        internal static extern void GetDomainSocketSizes(out int pathOffset, out int pathSize, out int addressSize);
```

Comment:
Ok, got some good clarification from @stephentoub over at https://github.com/dotnet/corefx/issues/10981, and I'll replace this call with hard coded values instead of using a call into a private extern wrapper.